local/bin/5-upload-and-privilege-app

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash -e
+APP_NAME="hello-world"
```

Comment:
I think we may want to split this up a little, partly because a few of the steps could be performed by different personas (installing the app vs loading policy). Maybe `start-and-bind-app`, `entitle-app`, and `start-app`? I'd be interested to know if you think that's too granular.

Reply:
I did something like what you suggested.